packages/core/directives/control.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, ElementRef, Input, OnChanges, SimpleChanges, ViewEncapsulation} from '@angular/core';
+import {GoogleMapsAPIWrapper} from '../services/google-maps-api-wrapper';
+import {ControlPosition} from '../services/google-maps-types';
+
+/**
+ * AgmMapControl allows to add a custom control to the map
```

Comment:
Please add a link to this site: https://developers.google.com/maps/documentation/javascript/controls?hl=en#CustomPositioning